core/client/hdfs/src/main/java/alluxio/hadoop/HadoopUtils.java

```diff
@@ -184,5 +189,30 @@ private static void setConfigurationFromSystemProperty(Configuration configurati
     }
   }
 
+  /**
+   * @param alluxioURI Alluxio uri
+   * @return corresponding Hadoop Path instance
+   */
+  public static Path toPath(AlluxioURI alluxioURI) {
+    return new Path(alluxioURI.toString());
+  }
+
+  /**
+   * @param status Hadoop file status
+   * @return corresponding Alluxio uri status instance
+   */
+  public static URIStatus toUriStatus(FileStatus status) {
+    // FilePath is a unique identifier for a file, however it can be a long string
+    // hence using md5 hash of the file path as the identifier in the cache.
+    FileInfo info = new FileInfo();
+    info.setFileIdentifier(md5().hashString(status.getPath().toString(), UTF_8).toString());
```

Comment:
Should we use file id here?

Reply:
i don't think so. A HDFS file shall not have a fileID as fileID is alluxio generated